framework/src/utils/Conversion.C

```diff
@@ -517,6 +517,92 @@ stringify(ElementType t)
   }
 }
 
+std::string
```

Comment:
https://mooseframework.inl.gov/docs/doxygen/libmesh/namespacelibMesh_1_1Utility.html#aa99a42a5efab878469e7f58e5c033335

with `libMesh::enum_to_strong<ElemType>(t)`

Reply:
this is done already in devel. Just need a next merge to rebase and use that code

Reply:
done